packages/server/src/koaJsonRpc/lib/utils.ts

```diff
@@ -0,0 +1,26 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import type { Server } from 'http';
+
+export function setServerTimeout(server: Server): void {
+  const requestTimeoutMs = parseInt(process.env.SERVER_REQUEST_TIMEOUT_MS ?? '60000');
+  server.setTimeout(requestTimeoutMs);
```

Comment:
We might need to do something about notifying users about the requests exceed limit timeout. Currently, the request just suddenly drops if exceeds the limit. The log also needs to be updated to log out a timeout notification

```bash
[2024-05-24 21:23:48.361 +0000] TRACE (relay-eth/16504 on admins-MBP-4.lan): [Request ID: 5747416b-a24d-48f1-a11b-ebd73ddc1ef6] blockNumber()
[2024-05-24 21:23:48.975 +0000] DEBUG (mirror-node/16504 on admins-MBP-4.lan): 5747416b-a24d-48f1-a11b-ebd73ddc1ef6 [GET] blocks?limit=1&order=desc 200 12 ms
[2024-05-24 21:23:48.975 +0000] TRACE (localLRUCache/16504 on admins-MBP-4.lan): [Request ID: 5747416b-a24d-48f1-a11b-ebd73ddc1ef6] caching eth_block_number:"0x14af" for 1000 ms
[2024-05-24 21:23:48.975 +0000] INFO (rpc-server/16504 on admins-MBP-4.lan): [Request ID: 5747416b-a24d-48f1-a11b-ebd73ddc1ef6] [POST]: eth_blockNumber 200 615 ms
```

<img width="1229" alt="image" src="https://github.com/hashgraph/hedera-json-rpc-relay/assets/66233296/cfc4ed89-68c0-48d8-9614-ad35b6bf972d">

Reply:
That's the expected response on a node server timeout.  Agreed that perhaps a more descriptive response facilitating graceful handling of the timeout can be explored, but is it in the scope of this change.